objmodel/chapter.md

```diff
@@ -1,86 +1,86 @@
-
+<!-- English spelling: "behaviour" line 46 -->
```

Comment:
Yes, that's a note to myself about spelling. I'm afraid you let quite a few stray "u"s slip in in "behaviour", and -ize and object-oriented are quite acceptable in British English. The OED is quite forgiving. :)